mlrun/db/httpdb.py

```diff
@@ -3989,10 +3996,11 @@ def store_alert_config(
     ) -> AlertConfig:
         """
         Create/modify an alert.
+
         :param alert_name: The name of the alert.
         :param alert_data: The data of the alert.
-        :param project: the project that the alert belongs to.
-        :return: The created/modified alert.
+        :param project:    The project that the alert belongs to.
+        :return:           The created/modified alert.
```

Comment:
I see the `param`s and `return` sometimes has a blank line between them and sometimes not.
What is the correct format and is it displayed differently?

Reply:
The blank line (or not) does not affect the display.